src/patternfly/components/Select/examples/select-single-top-expanded-example.hbs

```diff
@@ -0,0 +1,3 @@
+{{#> select id="select-single-top-expanded" select-menu-modifier="pf-m-top" select--IsExpanded="true"}}
```

Comment:
```suggestion
{{#> select id="select-single-top-expanded" select-menu--modifier="pf-m-top" select--IsExpanded="true"}}
```

Reply:
shouldn't it be `select--modifier`?

Reply:
@boaz0 oh nope, since we're adding `.pf-m-top` to `.pf-m-select__menu`, not `.pf-c-select`.